lib/md051.js

```diff
@@ -64,6 +64,34 @@ function unescapeStringTokenText(token) {
     .join("");
 }
 
+/**
+ * Gets the line number string from a fragment.
+ *
+ * @param {string} fragment Fragment string.
+ * @returns {string} Line number string.
+ * @example getLineNumberStringFromFragment("#L50") // 50
+ */
+function getLineNumberStringFromFragment (fragment) {
+  return fragment.slice(2)
+}
+
+/**
+ * Checks if a string is a valid integer.
+ *
+ * Using `Number.parseInt` combined with `Number.isNaN` will not be sufficient enough because `Number.parseInt("1abc", 10)` will return `1` (a valid number) instead of `NaN`.
+ *
+ * @param {string} value String to check.
+ * @returns {boolean} `true` if string is a valid integer, `false` otherwise.
+ * @example isValidIntegerString("1") // true
+ * @example isValidIntegerString("45") // true
+ * @example isValidIntegerString("1abc") // false
+ * @example isValidIntegerString("1.0") // false
+ */
+function isValidIntegerString (value) {
```

Comment:
I don't know if this function should be there?
Would it be good to move it elsewhere and do some proper unit testing specifically for this function only?